mypy/test/testgraph.py

```diff
@@ -42,6 +44,8 @@ def _make_manager(self) -> BuildManager:
         options = Options()
         fscache = FileSystemCache()
         search_paths = SearchPaths((), (), (), ())
+        stdout = StringIO()
+        stderr = StringIO()
```

Comment:
This essentially silences the output from the test, which may not be what's desired. (At least I don't see any code that captures the contents of `stdout/stderr`.) Why not use `sys.stdout/stderr`?

Reply:
I think this was accidentally left over from my initial attempt to make the arguments mandatory and change all the callsites before I noticed that was so much stuff. I'll remove it.